diamm/templates/website/contribution/contributions_made_list.jinja2

```diff
@@ -0,0 +1,8 @@
+<h2>Contributions</h2>
+<div class = "sixteen columns">
+    <ul class="contribution-list">
+        {% for contribution in contributions %}
+            <li>{{ contribution.contributor }}: {{ contribution.summary }}. {{ contribution.updated }}</li>
```

Comment:
If you only want the date, you can create a custom filter to format a datetime field to just show the date, without needing to change the model (as i commented above).